src/firebaseConfig.ts

```diff
@@ -4,10 +4,11 @@
 // the schema/ directory. After changing this file you will need to run
 // 'npm run generate:json-schema' to regenerate the schema files.
 //
-
 import { RequireAtLeastOne } from "./metaprogramming";
 import type { HttpsOptions } from "firebase-functions/v2/https";
 import { IngressSetting, MemoryOption, VpcEgressSetting } from "firebase-functions/v2/options";
+// Sourced from - https://docs.microsoft.com/en-us/javascript/api/@azure/keyvault-certificates/requireatleastone?view=azure-node-latest
+import { Emulators } from "./emulator/types";
```

Comment:
This line is causing the errors, though i am not entirely clear why. I think importing it is causing the strictNullChecks to in `npm run generate:json-schema to be applied across the entire codebase, which fails/